src/app/containers/Ad/Amp/index.test.jsx

```diff
@@ -50,16 +65,26 @@ describe('AMP Ads', () => {
   });
 
   describe('Assertions', () => {
-    it('should not render ad placeholder in UK', () => {
+    it('should not render ad placeholder in UK when adPlaceholder toggle is false', () => {
```

Comment:
```suggestion
    it('should not render ad placeholder in UK when showAdPlaceholder in service config is false', () => {
```
Just a suggestion since the toggle has been removed?